Makefile

```diff
@@ -4,8 +4,13 @@ KERNEL_DIR	?= /usr/src/linux
 
 CC		:= $(CROSS_COMPILE)gcc
 KERNEL_INCLUDE	:= -I$(KERNEL_DIR)/include -I$(KERNEL_DIR)/arch/$(ARCH)/include
-CFLAGS		:= -W -Wall -g $(KERNEL_INCLUDE)
-LDFLAGS		:= -g
+# For 720p
+#CFLAGS			:= -W -Wall -g $(KERNEL_INCLUDE)
+#LDFLAGS		:= -g
+
+# For 1080p
+CFLAGS          := -W -Wall -O3 $(KERNEL_INCLUDE)
```

Comment:
Do these options need to be switched back for 720p? (I don't see how they'd make a big difference either way...).

Reply:
I'm going to leave them in, they shouldn't affect 720/1080, really.